metrics/ci_integrations/test/source/github_actions/adapter/github_actions_source_client_adapter_test.dart

```diff
@@ -288,8 +317,9 @@ void main() {
     );
 
     test(
-      ".fetchBuilds() returns no more than the GithubActionsSourceClientAdapter.fetchLimit builds",
+      ".fetchBuilds() returns no more than the given initial sync limit number of builds",
```

Comment:
```suggestion
      ".fetchBuilds() returns no more than the given fetch limit number of builds",
```
Please, apply this suggestion to all similar tests.